app/assets/scripts/components/export-button.js

```diff
@@ -19,7 +19,14 @@ class ExportButton extends React.Component {
 
   componentWillReceiveProps (newProps) {
     if (this.props.csv.fetching && !newProps.csv.fetching && !newProps.csv.error) {
-      const encodedUri = encodeURI('data:text/csv;charset=utf-8,' + newProps.csv.data);
+      var thisdata = newProps.csv.data.replace(/\n/,'¤𝛼¤ß¤');
+      var firstLine = thisdata.split('¤𝛼¤ß¤')[0];
+      var moreLines = thisdata.split('¤𝛼¤ß¤')[1];
```

Comment:
This creates two potentially very large arrays when you could simply create one. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice